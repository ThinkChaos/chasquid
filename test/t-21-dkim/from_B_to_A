From: user-b@srv-b
To: user-a@srv-a
Subject: Feliz primavera!

Espero que florezcas feliz!
