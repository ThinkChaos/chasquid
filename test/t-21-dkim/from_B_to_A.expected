From user-a@srv-a
Authentication-Results: srv-a
	;spf=none (no DNS record found)
	;dkim=pass  header.b=*
DKIM-Signature: v=1; a=ed25519-sha256; c=relaxed/relaxed;
	d=srv-b; s=sel77; *
	h=from:subject:to:from:subject:date:to:cc:message-id;
	bh=*
	b=*
	  *
From: user-b@srv-b
To: user-a@srv-a
Subject: Feliz primavera!

Espero que florezcas feliz!
